test/fit_tests/README.md

```diff
@@ -67,6 +68,8 @@ All FIT tests can be run from the wrapper 'run_tests.py':
       -template TEMPLATE  path or URL link to OVA template or OnRack OVA, default
                           from global_config.json
       -xunit              generates xUnit XML report files
+      -numvms NUMVMS      number of virtual machine for deployment on specified
+                          stack
```

Comment:
typo - number of virtual machines    (vs machine)  
is there a default?  ie. default is 1?